Makefile

```diff
@@ -10,7 +10,7 @@ MAN_FILES = $(wildcard man/*.md)
 MAN_PAGES = $(patsubst man/%.md,man/%,$(MAN_FILES))
 
 PROJECT = github.com/codeclimate/test-reporter
-VERSION ?= 0.6.2
+VERSION ?= netcgo
```

Comment:
Is this really the default VERSION we want set all the time?

Reply:
no sorry that was an accident